js/user_migration-admin-settings.js.LICENSE.txt

```diff
@@ -0,0 +1,34 @@
+/*!
+ * The buffer module from node.js, for the browser.
```

Comment:
was this added on purpose ? strange that it's not in the compiled assets ?

Reply:
Added by `npm run build` same as seen in https://github.com/nextcloud/app-tutorial/blob/1dd70d3f63002745c78b1992e66a284b283252ec/js/notestutorial-main.js.LICENSE.txt which was used as a base for this app